include/core-image-bsp.inc

```diff
@@ -0,0 +1,31 @@
+# Basic packages/tools for RZ/G2L-SBC
```

Comment:
@tiennguyenuh ,

The name of this file does not fit the situation. In this case, we want to include some tool packages for core-image-minimal but the .inc file is named bsp which is for core-image-bsp?

Can you rename it to something like "core-image-renesas-base.inc"
https://github.com/Renesas-SST/meta-renesas/blob/dunfell/rz-sbc/meta-rz-common/include/core-image-renesas-base.inc

Reply:
Updated in the commit: https://github.com/Renesas-SST/meta-renesas/compare/bee65d49d368f5e35779eed10e83f6a9a50ad5a3..de7186edbd510d4722d284e38443ce313d4e89a1#diff-a930bba390b1341d1e579965c6ee6cc500181540e88745936a4353609cf186b6R1